### PR TITLE
Feat: define Jekyll collection for Press

### LINKED ISSUE #106: Press Release Jekyll collection
Based on this sheet: https://docs.google.com/spreadsheets/d/1xJpgr8JBG9TgZF2mHB8Vjnhu453No4dHoNjFIhb0FQw/edit#gid=0

See `Column C - Hosted on (calitp.org)` and filter to `Press Page` for the list to use for this collection.

## Tasks

- [x] define collection in Jekyll
- [x] define collection items based on existing Press Releases (see above)
- [x] define template to render list of Press Releases
- [x] define template to render individual Press Release

### PR DESCRIPTION
**Pull request by thekaveman**
Closes #106 

Note I moved the styling of the Press Release list on the landing page to #107 and split out the styling of the Press Release page to #140.

### CODE REVIEW

**Review by machikoyasuda — APPROVED** [review]
LGTM!

Just wondering - Is the page.external/script_redirect feature... a government requirement or something?

**Comment by thekaveman** [comment]
>Is the page.external/script_redirect feature... a government requirement or something

😅 Ha! Not sure what you mean by government requirement!?

We have at least one Press Release listed here that Cal-ITP didn't write, we're just linking to an external website. On the Press landing page, we use that external link so users should go straight there when clicking the link.

But since `output: true` for the collection, Jekyll still generates a page for that external press release item, e.g. `/press/cal-itp-payments-data-dashboard.html`.

In the very small chance that someone browsed to that page (like they have to know the exact URL and go there), it would look broken since the content is all empty. Hence the redirect for `external` Press Releases.

**Comment by thekaveman** [comment]
@machikoyasuda I'm going to rebase to get your navbar in here.

**Comment by machikoyasuda** [comment]
@thekaveman OOoooOohh, I see!!!!!!

**Comment by machikoyasuda** [comment]
image
Nav looking good.

**Review by machikoyasuda — DISMISSED** [review]
LGTM!

I tested this redirect link: https://deploy-preview-118--cal-itp-website.netlify.app/press/cal-itp-payments-data-dashboard

**Comment by angela-tran** [comment]
I've been reviewing this - should be done soon

File: src/_layouts/press.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <a href="/press">Press</a>
+    <h1 class="h2">{{ page.heading }}</h1>
+    {% if page.lead %}
+      <p>{{ page.lead }}</p>
+    {% endif %}
+    <hr/>
+    <p class="immediate-release">For immediate release</p>
+    <p>{{ date | append: location | append: "—" | append: page.intro }}</p>
+    {{ page.content }}
+    <p class="end">###</p>
```
Comment (angela-tran) [inline src/_layouts/press.html:26]: What is this for?

**Review by angela-tran — DISMISSED** [review]
Left comments for small things that are not blockers

File: src/_layouts/redirect.html [hunk 2]
```diff
@@ -3,26 +3,24 @@
...
-            <h1>Redirecting...</h1>
-            <a href="{{ page.redirect_to }}">Click here if you are not redirected.<a>
-            <script>location = '{{ page.redirect_to }}'</script>
-          </section>
+      <section id="deck">
+        <section>
+          {% include script_redirect.html location=page.redirect_to %}
         </section>
+      </section>
     </article>
```
[the remark this answers is not in the document]
Comment (machikoyasuda) [reply src/_layouts/redirect.html:23]: This could be part of #107

[the remark this answers is not in the document]
Comment (thekaveman) [reply src/_layouts/redirect.html:23]: I'll clean this up here since I already touched that file

File: src/_layouts/press.html (hunk 1 above)
Comment (thekaveman) [reply src/_layouts/press.html:26]: It's usually at the bottom of every Press Release, e.g. https://www.calitp.org/assets/Cal-ITP.Benefits.PressRelease.220921.pdf
image

I see that it didn't make it into the design, so we can probably omit from the template.

Comment (angela-tran) [reply src/_layouts/press.html:26]: Oh I see. Thanks for linking that example

**Review by angela-tran — APPROVED** [review]

**Pull request merged**